src/layouts/Dashboard/components/ReviewRequestModal/RequestOverview/RequestOverview.tsx

```diff
@@ -0,0 +1,335 @@
+    columnHelper.display({
+      id: "filter",
+      header: () => (
+        <Th w="3rem">
+          <IconButton
+            aria-label="sort by file type"
+            icon={<BiFilterAlt />}
+            variant="link"
+          />
+        </Th>
+      ),
+      cell: (props) => getIcon(props.row.original.type),
+    }),
+    columnHelper.accessor((row) => `${row.name} ${row.path}`, {
+      id: "itemName",
+      cell: ({ row }) => (
+        <ItemName name={row.original.name} path={row.original.path} />
+      ),
+      header: () => (
+        <Th textTransform="capitalize">
+          <Text textStyle="subhead-2" textColor="text.label">
+            Item name
+          </Text>
+        </Th>
+      ),
+    }),
+    columnHelper.accessor(
+      (row) => `${row.lastEditedBy} ${row.lastEditedTime}`,
+      {
+        id: "lastEdited",
+        header: () => (
+          <Th textTransform="capitalize" w="12.5rem">
+            <HStack spacing="0.25rem">
+              <Text textStyle="subhead-2" textColor="text.label">
+                Last edited
+              </Text>
+              <IconButton
+                icon={<BiSort fontSize="1rem" />}
+                aria-label="view file changes"
+                variant="link"
+              />
+            </HStack>
+          </Th>
+        ),
+        cell: ({ row }) => (
+          <LastEditedMeta
+            lastEditedBy={row.original.lastEditedBy}
+            lastEditedTime={row.original.lastEditedTime}
+          />
+        ),
+      }
+    ),
+    columnHelper.display({
+      id: "actions",
+      header: () => <Th w="10rem" />,
+      cell: ({ row }) => (
+        <HStack spacing="0.25rem">
+          <Link href={row.original.url}>
+            <IconButton
+              icon={<BiGitCompare />}
+              aria-label="view file on staging"
+              variant="link"
+            />
+          </Link>
+          <IconButton
+            icon={<BiShow />}
+            aria-label="view file changes"
+            variant="link"
+          />
+        </HStack>
+      ),
+    }),
+  ]
+
+  const table = useReactTable({
+    data: items,
+    columns,
+    getCoreRowModel: getCoreRowModel(),
+  })
+
+  return (
+    <VStack spacing="1rem" mt="1rem">
+      <Flex w="100%">
+        <Box>
+          <Text textStyle="subhead-1" textColor="text.body">
+            <b>{items.length}</b> item(s) for review
+          </Text>
+          <Text textStyle="caption-2" textColor="text.placeholder">
+            Items below have been identified to have changes
+          </Text>
+        </Box>
+        <Spacer />
+        <Searchbar
+          ref={inputRef}
+          onSearchIconClick={handleExpansion}
+          isExpanded={isExpanded}
+          onSearch={() => {
+            console.log("hi")
+          }}
+          onBlur={handleCollapse}
+        />
+      </Flex>
+      <Box w="100%" borderWidth="1px" borderRadius="8px" borderColor="gray.100">
+        <TableVirtuoso
```

Comment:
I'm not sure why the scrollbar goes all the way to the top. I think should stop before the header row.

<img width="720" alt="Screenshot 2022-09-27 at 10 20 09" src="https://user-images.githubusercontent.com/27919917/192416313-57af5199-3fd3-488d-b86a-5aed750a4fea.png">

Reply:
i think this is from the library as this is `sticky` - i can check w/ @NatMaeTan to see if this is an issue but this should be relatively minor

Reply:
@NatMaeTan nodded her head in approval so i think this is ok.